components/logos/Adidas.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * @description Logo for Adidas
```

Comment:
Add `props.className` param in jsdocs. Do it for every component.

Reply:
done for all

Reply:
ohh missed the logos.